supervisely/nn/inference/inference.py

```diff
@@ -985,6 +985,19 @@ def get_preparing_progress(response: Response, request: Request):
             inference_request = self._inference_requests[inference_request_uuid].copy()
             return inference_request["preparing_progress"]
 
+        @server.post("/deploy_nn_serving")
+        def deploy_nn_serving(response: Response, request: Request):
```

Comment:
1. run app via api (with autostart=False)
2. api to deploy model checkpoint -> checkpoint has to be downloaded to directory -> deploy
3. API to stop deployed model and revert to initial state (like after 1)